packages/callout-block/src/CalloutBlock.tsx

```diff
@@ -90,15 +93,20 @@ export const CalloutBlock: FC<BlockProps> = ({ appBridge }) => {
                         color={textColor}
                     />
                 )}
-                <RichTextEditor
-                    onTextChange={onTextChange}
-                    onBlur={onTextChange}
-                    readonly={!isEditing}
-                    border={false}
-                    value={blockSettings.textValue}
-                    placeholder="Type your text here"
-                    designTokens={useCalloutColors(designTokens, textColor)}
-                />
+                {!isEditing ? (
+                    <div data-test-id="rte-content-html" dangerouslySetInnerHTML={{ __html: html }} />
+                ) : (
+                    <RichTextEditor
+                        data-test-id="rich-text-editor"
```

Comment:
doesn't RTE has this data-test-id built-in?

Reply:
Holymoly, good point, thanks!

Reply:
Well should we really rely on "external" test ids for tests?

Reply:
The `data-test-id` prop on the component currently doesn't seem to have an effect anyway, so I would keep it as it for the moment. If we update fondue in the future and the 'external' test id changes, we will notice when the tests fail. :)